utilities/blob_db/blob_file.cc

```diff
@@ -7,6 +7,7 @@
 #include <chrono>
 #include <cinttypes>
 #include <memory>
+#include <stdio.h>
```

Comment:
Shall we do `#include \<cstdio\>` instead, so that all header files are c++ style, and the functions provided by stdio go into `std` namespace (no globals)? 
I see in our code we use both `stdio.h` and `cstdio`, but it would be good to stick to one type (my personal preference is c++ style header files).  (larger change for a later time ... I can take it up).